content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -398,7 +461,7 @@ To control the volume of Nginx traces that are sent to Datadog, specify a sampli
 }
 ```
 
-If no value is specified, the [Datadog Agent calculated sampling rates][7] (10 traces per second per Agent) are applied.
+If no sample rate is specified, the [Datadog Agent calculated sampling rates][7] (10 traces per second per Agent) are applied.
```

Comment:
Does it also apply to Nginx module? As this section falls under the Nginx with opentracing module I would assume it is only relevant for this case

Reply:
`### Nginx Sampling` is an `h3` section, the same as the OpenTracing and Datadog module sections.  So, there is one section about getting the Datadog module, another about getting the OpenTracing module, and then a third about configuring sampling for them.

This is possibly confusing.  What might be better?

Reply:
I am wrong.  Will fix.